### PR TITLE
New user when login

### PR DESCRIPTION
**Pull request by CHOCOLVT**
still need to improve UX with right colors and small details

### CODE REVIEW

**Comment by JorisMonnet** [comment]
I approved on code climate, add some tests for diff-coverage and it will be perfect

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,55 +1,56 @@
...
+
+            <!-- The meta-data tag is required if you support API level 15 and lower -->
+            <meta-data
+                android:name="android.support.PARENT_ACTIVITY"
+                android:value=".ui.MainMenuActivity" />
+        </activity>
+        <activity
+            android:name=".ui.tutorial.TutorialActivity"
+            android:exported="false" />
+        <activity
```
Comment (marluxiaboss) [inline app/src/main/AndroidManifest.xml:30]: You should define parents in the manifest for navigation.

**Review by JorisMonnet — APPROVED** [review]

File: app/src/main/java/ch/epfl/sdp/blindwar/ui/SplashScreenActivity.kt [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                // new user: 1st signIn
+                return Intent(activity, NewUserActivity::class.java)
+            } else {
+                /*
+            - should we update the online database with the local cache here ?
+             */
+                return Intent(activity, MainMenuActivity::class.java)
+            }
+        } else {
+            // Sign in failed. If response is null the user canceled the
```
Comment (marluxiaboss) [inline app/src/main/java/ch/epfl/sdp/blindwar/ui/SplashScreenActivity.kt:94]: Weird comments

**Review by marluxiaboss — APPROVED** [review]
Some doc is needed.

**Pull request merged**